components/contextualhelp/index.css

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-ContextualHelp {
+  /* Layout Variables */
+  --spectrum-contextual-help-minimum-width: var(--spectrum-contextual-help-minimum-width);
+  --spectrum-contextual-help-padding: var(--spectrum-spacing-400);
+  --spectrum-contextual-help-content-spacing: var(--spectrum-spacing-100);
+  --spectrum-contextual-help-link-spacing: var(--spectrum-spacing-300);
+
+  /* Typography Variables */
+  /* TODO: these need to be changed to the contextual-help tokens when available */
+  --spectrum-contextual-help-title-size: var(--spectrum-heading-size-xs);
+  --spectrum-contextual-help-body-size: var(--spectrum-body-size-s);
+
+
+
+  /* Mobile styling */
+  @media screen and (max-width: 700px) {
+    --spectrum-contextual-help-content-spacing: var(--spectrum-spacing-200);
+  }
```

Comment:
I am assuming this is handling the desktop to mobile difference for these tokens? 
![Screenshot 2023-03-21 at 12 03 05 PM](https://user-images.githubusercontent.com/63808889/226700966-527d709f-4a6d-41e4-b679-98800de5e24d.png)

I have noticed the designers use Mobile and Desktop to refer to Large vs Medium scale and not the actual screen size.

Reply:
Ok this would make sense. It was one of my questions as I worked on this what the designation was. @castastrophe do you know if that's pretty typical across the board for Desktop and Mobile to be Large and Medium scale?

Reply:
Yeah - the desktop & mobile styles map to platform sizes large and medium respectively. The foundational variables will be updated when the scale is altered so no need to set that here. If you do need to set a custom variable against a platform size, I'd imagine you could use: `.spectrum--large &`?